web_src/js/markup/math.js

```diff
@@ -15,18 +15,28 @@ export async function renderMath() {
     import(/* webpackChunkName: "katex" */'katex/dist/katex.css'),
   ]);
 
+  const MAX_CHARS = 1000;
```

Comment:
Do we want a setting for this like with Mermaid?

https://github.com/go-gitea/gitea/blob/5a76759c9ca7a1ff49b9297b57cde1bd1586c4fe/web_src/js/markup/mermaid.js#L5

Reply:
I'm not a fan of this addition.
Isn't 1000 chars too restrictive?
That's only four tweets, and there are many comments larger than that.
Especially as that is per comment instead of per recognized latex string.

As I see it, we can either remove it, or we choose the value that GitLab for example uses.

Reply:
GitLab uses 1000. Why do you think it's the whole comment? `source` is just the `<code>` block with the math code.

Reply:
Ah, I should have expanded the code a little.
The `el.textContent` looked to me like it would be the whole comment, not the math block.